js/main.js

```diff
@@ -6,3 +6,51 @@ const isValidLengthCommit= (string = '') => string.length <= MAX_COMMIT_LENGTH;
 
```

Comment:
```suggestion
const isValidLengthCommit = (string = '') => string.length <= MAX_COMMIT_LENGTH;
```

Пропущен пробел на 5й строке перед знаком "=".
Пожалуйста, относись повниматльнее к форматированию кода, это важная составляющая любой работы - аккуратность.

Разумеется её можно автоматизировать, настроив свой редактор на это.
Если у тебя VSCode, то могу порекомендовать такую небольшую инструкцию о том как это можно сделать: [VSCode_setting_part_2.md#плагины](https://gist.github.com/LenaVladi/4bd31c81df7421b3347aa7eba4ab5c45#%D0%BF%D0%BB%D0%B0%D0%B3%D0%B8%D0%BD%D1%8B)